### PR TITLE
Feature/add locale support

### LINKED ISSUE #18: Add Internalization with I18n
With I18n we can provide translation for erros and model attributes.

### PR DESCRIPTION
**Pull request by nemaberci**
Adding language selector to navigation bar (may need redesign), adding some example code to kitchens / index.

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -9,7 +9,18 @@ require("@rails/activestorage").start()
...
+        console.log(event.target.value);
+        if (document.location.search.length !== 0) { // If there is already a querystring
+            console.log("new: " + document.location.search.replace(/locale=[a-z]+/, 'locale=' + event.target.value));
+            document.location.search = document.location.search.replace(/locale=[a-z]+/, 'locale=' + event.target.value);
+        } else {
+            console.log("new: " + '?locale=' + event.target.value);
+            document.location.search = '?locale=' + event.target.value;
+        }
+    });
+});
```
Comment (triszt4n) [inline app/javascript/packs/application.js]: Let's clear the code of unnecessary console.logs and semicolons.

**Review by triszt4n — CHANGES_REQUESTED** [review]
Nice implementation, everything neat and clean. :sparkles: 
Please have a look at my request below.
Once this PR is accepted we're gonna create an issue for designing the language selector later.

File: app/javascript/packs/application.js (hunk 1 above)
Comment (nemaberci) [reply app/javascript/packs/application.js]: Removed unnecessary comments

**Review by SepsiLaszlo — CHANGES_REQUESTED** [review]
Sorry, I wasn't to specific about the issue. :(
I think this application does not need to support English.
The issue targeted the translation of the model attribute names, the validation error messages and other dynamically generated thing to things to Hungarian. 
Static stings aren't necessary to use I18n.

I think it would be enough for this PR to generate translation_hu.yml with [i18n_generators](https://github.com/amatsuda/i18n_generators) and set the `default_locale` to hungarian.

I'm terrible sorry for the unclear issue, I'll try to describe them more properly. Please contact me If something is not specific enough.

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -14,20 +14,20 @@
...
-              %svg.fill-current.h-4.w-4.text-red-600{:role => "button", :viewbox => "2 2 16 16", :xmlns => "http://www.w3.org/2000/svg"}
-                %title Close
-                %path{:d => "M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"}
-            .flex
-              .py-1
-                %svg.fill-current.h-5.w-5.text-teal-500.mr-4{:viewbox => "0 0 20 20", :xmlns => "http://www.w3.org/2000/svg"}
-                  %path{:d => "M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z"}
-              %div
-                %p.text-md= flash[:notice]
+        - # notice box
```
Comment (bbucsy) [inline app/views/layouts/application.html.haml]: This indentation part breaks the page on my machine:
konyha_break

**Review by bbucsy — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.haml (hunk 2 above)
Comment (nemaberci) [reply app/views/layouts/application.html.haml]: I made a mistake while merging master into the branch. Fixing it in next commit.

**Review by SepsiLaszlo — APPROVED** [review]
Nice work! 

Thank you for making the requested changes!

**Review by bbucsy — CHANGES_REQUESTED** [review]
It kind of breaks Faker in `seeds.rb`

**Review by triszt4n — APPROVED** [review]

**Review by bbucsy — APPROVED** [review]
Noice

**Pull request merged**